CHANGELOG.md

```diff
@@ -48,7 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- API for custom server messages now uses `server_event::serialize_with`  and `server_event::deserialize_with`. For more details see the example in the docs.
```

Comment:
Didn't mean to make this change, guess my editor auto-formatted it?

Reply:
It's fine, keep it.